Aztec/Classes/Public/TextKit/TextAttachment.swift

```diff
@@ -23,9 +26,16 @@ public class TextAttachment: NSTextAttachment
 
     private var glyphImage: UIImage?
 
-    /// Designed Initializer
+    public var imageProvider: TextAttachmentImageProvider?
+    public var fetchingHappening: Bool = false
```

Comment:
Maybe change this to `isFetchingImage` or something similar?

`fetchingHappening` sounds a bit non-standard.